ipaserver/install/adtrustinstance.py

```diff
@@ -345,9 +345,14 @@ def __add_rid_bases(self):
 
             # Abort if RID bases are too close
             local_range = ranges_with_no_rid_base[0]
-            size = local_range.single_value.get('ipaIDRangeSize')
+            try:
+                size = int(local_range.single_value.get('ipaIDRangeSize'))
```

Comment:
How about when `'ipaIDRangeSize'` is unset (`None`)? Should we rather skip comparison `if abs(self.rid_base - self.secondary_rid_base) < size:` than fail hard?